clippy_lints/src/methods/mod.rs

```diff
@@ -2432,7 +2432,7 @@ fn lint_find_map<'a, 'tcx>(
     }
 }
 
-/// lint use of `filter().map()` for `Iterators`
+/// lint use of `filter().map().map()` for `Iterators`
```

Comment:
Wasn't it meant to be `filter_map().map()`?

Reply:
facepalm... yes, that's right.